base/linalg/dense.jl

```diff
@@ -112,6 +112,43 @@ true
 isposdef(A::AbstractMatrix) = ishermitian(A) && isposdef(cholfact(Hermitian(A)))
 isposdef(x::Number) = imag(x)==0 && real(x) > 0
 
+"""
+    stride1(A) -> Int
+
+Returns the distance between successive array elements
+in units of element size.
+If the argument type is `Array`, the return value of `stride1` is always `1`.
```

Comment:
Perhaps unnecessary given the example? If you want to keep it I would suggest something like
```
`stride1` for `Array`s is always 1.
```

Reply:
Thank you for your comment.
I think same, too. So, I removed this example.